packages/slider/test/slider.test.ts

```diff
@@ -844,6 +844,53 @@ describe('Slider', () => {
         expect(el.variant).to.equal('tick');
         expect(el.getAttribute('variant')).to.equal('tick');
     });
+    it('renders div[class="fill"] when fill-start', async () => {
+        const el = await fixture<Slider>(
+            html`
+                <sp-slider max="100" fill-start min="0" value="10"></sp-slider>
+            `
+        );
+
+        await elementUpdated(el);
+        await nextFrame();
+        await nextFrame();
+        expect(el.values).to.deep.equal({ value: 10 });
+
+        const handle = el.shadowRoot.querySelector('.handle') as HTMLDivElement;
+        const handleBoundingRect = handle.getBoundingClientRect();
+        const position: [number, number] = [
+            handleBoundingRect.x + handleBoundingRect.width / 2,
+            handleBoundingRect.y + handleBoundingRect.height / 2,
+        ];
+        await sendMouse({
+            steps: [
+                {
+                    type: 'move',
+                    position,
+                },
+                {
+                    type: 'down',
+                },
+            ],
+        });
+
+        await elementUpdated(el);
+        await sendMouse({
+            steps: [
+                {
+                    type: 'move',
+                    position: [
+                        200,
+                        handleBoundingRect.y + handleBoundingRect.height + 100,
+                    ],
+                },
+            ],
+        });
+        await nextFrame();
+
+        expect(el.value).to.equal(24);
+        expect(el.shadowRoot.querySelector('.fill') as HTMLDivElement).to.exist;
```

Comment:
Probably a good idea to assert that `.fill` doesn't exist to get started with?

Reply:
class fill will be there when fill-start or fill-start with value is there with width as 0% when only fill-start

Reply:
Then a presence check is likely not enough to actually guarantee that the role that element is playing is complete to the situation in question.